### PR TITLE
Serpentid queen, the medical support

### PR DESCRIPTION
**Pull request by Rayfod**
The Serpentid queen was using their entire point catalog of various medical-related skills
I also made their points the exact same as the vox biotechnician
Skærmbillede 2022-03-13 222331

### CODE REVIEW

File: maps/away/ascent/ascent_jobs.dm [hunk 1]
```diff
@@ -241,9 +241,11 @@
...
 	min_skill = list(SKILL_EVA = SKILL_ADEPT,
 					SKILL_HAULING = SKILL_ADEPT,
-					SKILL_COMBAT = SKILL_ADEPT,
-					SKILL_WEAPONS = SKILL_ADEPT,
-					SKILL_MEDICAL = SKILL_BASIC)
+					SKILL_COMBAT = SKILL_BASIC,
+					SKILL_WEAPONS = SKILL_BASIC,
+					SKILL_MEDICAL     = SKILL_EXPERT,
+					SKILL_ANATOMY     = SKILL_EXPERT,
+					SKILL_CHEMISTRY   = SKILL_ADEPT)
```
Comment (Cupax3) [inline maps/away/ascent/ascent_jobs.dm]: Pedantic, but if you align one set of skills, why not align the rest too?

**Comment by Rayfod** [comment]
My mistake

**Review by Cupax3 — DISMISSED** [review]
Code looks fine, lore/balance remains to be seen

**Comment by Rayfod** [comment]
My main reason for having it added, was because of their flavor text saying "You are a Monarch Serpentid Queen living on an independant Ascent vessel. Assist the Gyne in her duties and _tend to your Workers_."

> 410,757,864,350 DEAD ALATES — Today at 4:21 PM
> queens aren't literally meant to do tend to their workers but giving them an inclination to be surgeons isn't a bad thing
> [4:22 PM]
> they're supposed to be reliant on their adjuncts for protection anyway
> [4:22 PM]
> I don't have a problem with this

**Comment by Anton-Kr** [comment]
Nothing here lowers the queens overall points

**Review by Anton-Kr — CHANGES_REQUESTED** [review]
Lower overall points

**Comment by Rayfod** [comment]
```
SKILL_COMBAT = SKILL_BASIC,
SKILL_WEAPONS = SKILL_BASIC,
```
Their skills for combat was reduced to this, which makes their overall points 8 less
Do they need to be further reduced?

**Comment by Anton-Kr** [comment]
Correct me if my math is wrong: 

Combat going from trained to basic is a loss of 4 points.
WE going from trained to basic is a loss of further 4 points.

Medical going from basic to experienced is 12 points
Anatomy going from nada to experienced is 16 points.

Leading to a total gain of a whopping 20 points. I am not against the Queen gaining some skill points but this is far too many.

File: maps/away/ascent/ascent_jobs.dm [hunk 2]
```diff
@@ -242,9 +242,11 @@
 	outfit_type = /decl/hierarchy/outfit/job/monarch
```
Comment (Anton-Kr) [inline maps/away/ascent/ascent_jobs.dm:242]:
```suggestion
	outfit_type = /decl/hierarchy/outfit/job/monarch
	skill_points = 14
```

Comment (Anton-Kr) [reply maps/away/ascent/ascent_jobs.dm:242]: Mind you, I am fine with them getting _some_ amount of extra points, but this PR gives them 20 which is excessive.

**Comment by Rayfod** [comment]
Changes added as @Anton-Kr asked for

**Review by Anton-Kr — APPROVED** [review]

**Pull request merged**